src/node-win/watcher/watcher.unit.test.ts

```diff
@@ -27,7 +27,7 @@ describe('Watcher', () => {
       await sleep(50);
 
       // Assert
-      expect(getEvents()).toStrictEqual([{ event: 'addDir', path: syncRootPath }]);
+      getEvents().toStrictEqual([{ event: 'addDir', path: syncRootPath }]);
```

Comment:
Are you sure that the mix of hidden `expects` + hidden `calls` is also worth it? It seems not too intuitive. I had to check what getCalls was doing to know that the calls was being used there also